CHANGELOG.unreleased.md

```diff
@@ -11,12 +11,15 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Usage Report] Fix not updated report date [#4779](https://github.com/vatesfr/xen-orchestra/issues/4779) (PR [#4799](https://github.com/vatesfr/xen-orchestra/pull/4799))
```

Comment:
Please improve this message, changelog are not meant for devs but users and I fear it may not be clear enough.

Reply:
Better 👍